ee/clickhouse/models/entity.py

```diff
@@ -19,7 +19,11 @@ def get_entity_filtering_params(
     prop_filters = ""
     if with_prop_filters:
         prop_filters, params = parse_prop_clauses(
-            entity.properties, team_id, table_name=table_name, person_properties_column=person_properties_column
+            entity.properties,
+            team_id,
+            table_name=table_name,
+            person_properties_column=person_properties_column,
+            prepend=f"entity",
```

Comment:
Are there any other places where this might be required.

Maybe out of scope for this perhaps would would it be worth making this prepend non-optional so we are explicit everywhere.

Reply:
Might well be, it wouldn't hurt at least! 

The case it blows up is when multiple calls to `parse_prop_clauses` is made as part of one query and not every call has unique prepend.

Will think about this more later and solve in a different PR though.